profiles/Jonas-setup.yaml

```diff
@@ -1,31 +1,41 @@
 ---
-safe_room: 4377 # bnb
-# Hunting Settings
-
+attunement_rooms: #Non-justice attunement rooms near my safe room 
```

Comment:
Is there any risk that this could give away your safe room? Do attunement rooms need to also support being pulled from a UserVar to keep them out of your YAML?